lynx/patterns/link-in-bio-audio embed.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Title: Link in Bio with audio embed
```

Comment:
More of a general comment, but should the reference to Link in Bio be changed to Lynx, globally, since Link in Bio is a specific (non-a8c) product?

Reply:
Link in Bio is the naming convention used in the onboarding flow this will feed into, so I'm following that for individual patterns.